packages/core/db.ts

```diff
@@ -164,6 +164,19 @@ export interface ServiceJourneyPatternLinksTable {
     orderInSequence: number;
     fromSequenceNumber: string | null;
     toSequenceNumber: string | null;
+    routeLinkRef: string | null;
+}
+
+export interface RoutesTable {
+    id: Generated<number>;
```

Comment:
could we have a test written to see if it is getting the data from the xml?

Reply:
done